File: cgi-bin/second.pl
#!/usr/bin/perl

use strict;
use warnings;
use CGI;

my $cgi = CGI->new;
my %param = map { $_ => scalar $cgi->param($_) } $cgi->param() ;
print $cgi->header( -type => 'text/html' );
print "<h1>" . qq{PARAM:} . "</h1>";
for my $k ( sort keys %param ) {
    print "<div>" . join ": ", $k, $param{$k} . "</div>";
}

File: config/cgi-bin/second.py
#! /usr/bin/env python38

import html
import time
import cgi, cgitb 

cgitb.enable()

hit_count = 0

header = "Content-type: text/html\r\n\r\n"

date_string = time.strftime('%A, %B %d, %Y at %I:%M:%S %p %Z')

html = """
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>Current date</title>
</head>
<body>
  <p>
  Date: {0}
  </p>
  <p>
  Hit count: {1}
  </p>
</body>
</html>
""".format(html.escape(date_string), html.escape(str(hit_count)))

print(header + html)